tutorials/tutorial1.ipynb

```diff
@@ -10,11 +10,11 @@
     "\n",
     "In this tutorial we train an AtacWorks model to denoise the signal track and call peaks from aggregate single-cell ATAC-seq data derived from a small number of cells. We use the dsc-ATAC-seq dataset presented in reference (1) (Section \"AtacWorks enhances ATAC-seq results from small numbers of single cells\", also Supplementary Table 8). This dataset consists of single-cell ATAC-seq data from several types of human blood cells.\n",
     "\n",
-    "Note that all the AtacWorks models described in reference (1) are available to download (https://atacworks-paper.s3.us-east-2.amazonaws.com) and you may be able to use one of these instead of training a new model. To learn how to download and use an existing model, refer to [Tutorial 2](tutorial2.ipynb).\n",
```

Comment:
Let's move these instructions to the sphinx documentation, in the same place where we have the caveats and specific circumstances in which pre-trained models can be used. We can link to that documentation here.

Reply:
done